src/components/common/Header.tsx

```diff
@@ -1,12 +1,63 @@
+import { setIsAuthDialogOpen, setIsLoggedIn } from "@redux/actions/authActions";
+import { AuthSelectors } from "@redux/selectors/authSelectors";
 import styles from "@styles/components/common/Header.module.css";
+import { signOut, useSession } from "next-auth/react";
 import Image from "next/image";
+import { Button } from "primereact/button";
 import { Menubar } from "primereact/menubar";
-import { MenuItem } from "primereact/menuitem";
 import logo from "public/logo.png";
-import React from "react";
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { AuthDialogType } from "src/types/auth";
+import AuthDialog from "../auth/authDialog";
+const classNames = require("classnames");
```

Comment:
Use ES6 import syntax
```suggestion
import classNames from "classnames";
```